test/common/controllers/base-address.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const proxyquire = require('proxyquire');
+class StubBaseController {}
+
+StubBaseController.prototype.locals = sinon.stub();
+
+const BaseAddressController = proxyquire('../../../apps/common/controllers/base-address', {
+  './base': StubBaseController
+});
+
+describe('BaseAddressController', function() {
+
+  it('extends base-controller', ()=> {
+    BaseAddressController.prototype.should.be.an.instanceOf(StubBaseController);
+  });
+
+  it('has a locals, translateCategories, mapAddress & hasCategories methods', ()=> {
+    BaseAddressController.prototype.should.have.property('locals').that.is.a('function');
```

Comment:
I don't feel particularly strongly about it, but its just something I have always done